Sources/WebDriver/Requests.swift

```diff
@@ -462,6 +462,70 @@ public enum Requests {
         }
     }
 
+    // https://www.selenium.dev/documentation/legacy/json_wire_protocol/#sessionsessionidwindow
+    public enum SessionWindow {
+        public struct Post: Request {
+            public var session: String
+            public var name: String
+
+            public var pathComponents: [String] { ["session", session, "window"] }
+            public var method: HTTPMethod { .post }
+            public var body: Body { .init(name: name) }
+
+            public struct Body: Codable {
+                public var name: String
+            }
+        }
+
+        public struct Delete: Request {
+            public var session: String
+            public var name: String
+
+            public var pathComponents: [String] { ["session", session, "window"] }
+            public var method: HTTPMethod { .delete }
+            public var body: Body { .init(name: name) }
+
+            public struct Body: Codable {
+                public var name: String
+            }
+        }
+    }
+
+    // https://www.selenium.dev/documentation/legacy/json_wire_protocol/#sessionsessionidwindowwindowhandlesize
+    public enum SessionWindowHandleSize {
```

Comment:
On further reflection, I wonder if we should name this the `WindowSize` request?  Each request is implicitly tied to the session.  The URL format is `/session/:sid/window/:wid/size/:sz`.  So, `SessionWindowSize` or `WindowSize` makes more sense IMO.

Reply:
Ah yeah, good catch.

Reply:
I might be looking at the wrong docs but the URL format looks to be `/session/:sessionId/window/:windowHandle/size`

Reply:
You're correct-- @compnerd was trying to save typing (and size goes in the body, not the URL, as you've coded it).

Reply:
Size doesn't appear to be a passable parameter in either the URL or the body. You only post the width and height in the json body and calling `GET` will send those dimensions back. I could be misunderstanding something though.